src/test/kotlin/com/hedvig/underwriter/service/CreateContractsFromQuotesSavesContractIdAndContractIdTest.kt

```diff
@@ -115,7 +115,7 @@ class CreateContractsFromQuotesSavesContractIdAndContractIdTest {
         signServiceImpl.signQuoteFromRapio(
             quoteId,
             SignQuoteRequestDto(
-                Name("Mr Test", "Tester"), null, LocalDate.of(2020, 1, 1), "a@email.com"
+                Name("Mr Test", "Tester"), null, LocalDate.of(2020, 1, 1), "if", "a@email.com"
```

Comment:
null will be the commonly used use case right?
```suggestion
                Name("Mr Test", "Tester"), null, LocalDate.of(2020, 1, 1), null, "a@email.com"
```